packages/protocol/contracts/layer1/preconf/impl/PreconfWhitelist.sol

```diff
@@ -209,17 +218,18 @@ contract PreconfWhitelist is EssentialContract, IPreconfWhitelist {
 
     /// @dev The cost of this function is primarily linear with respect to operatorCount.
     function _getOperatorForEpoch(uint64 _epochTimestamp) internal view returns (address) {
-        if (_epochTimestamp < LibPreconfConstants.SECONDS_IN_EPOCH) {
-            return address(0);
-        }
+        uint256 timeShift = selectorEpochOffset * LibPreconfConstants.SECONDS_IN_EPOCH;
+        // We use the beacon root at or after ` _epochTimestamp - timeShift` as the random number to
+        // select an operator.
+        // selectorEpochOffset must be big enough to ensure a non-zero beacon root is
+        // available and immutable.
```

Comment:
Why does having big enough `selectorEpochOffset` ensure non-zero beacon root? In other words, isn't something like following enough?

```suggestion
        // selectorEpochOffset must be big enough to ensure that the beacon root is finalized and
        // cannot change due to reorg.
```